test/unit/http.test.ts

```diff
@@ -174,5 +174,75 @@ describe('httpClient', () => {
 
       expect(cleanParameters(parameters)).toStrictEqual(results)
     })
+
+    it('should properly clean parameters with object arrays for attributes', () => {
+      expect.hasAssertions()
+
+      const parameters = {
+        MarketplaceId: 'ATVPDKIKX0DER',
+        Destination: {
+          DeliveryChannel: 'SQS',
+          'AttributeList.member': [
+            {
+              Key: 'sqsQueueUrl',
+              // eslint-disable-next-line sonarjs/no-duplicate-string
```

Comment:
Let's not disable this linting rule, but instead put the value in a constant at the top of the file or relevant `describe` code block.

Reply:
Ohh you're right. I missed this, I used a quick fix with the intention of fixing it after making sure that my tests passed but forgot to go back to it